projects/plugins/jetpack/_inc/blogging-prompts.php

```diff
@@ -73,7 +72,7 @@ function jetpack_get_daily_blogging_prompts( $time = 0 ) {
 	}
 
 	$blog_id = \Jetpack_Options::get_option( 'id' );
-	$path    = '/sites/' . $blog_id . '/blogging-prompts?from=' . $day_before . '&number=10&_locale=' . $locale;
+	$path    = rawurldecode( '/sites/' . $blog_id . '/blogging-prompts?from=' . $day_before . '&number=10&_locale=' . $locale );
```

Comment:
Wrong function, and the right function won't do the right thing as you're doing it here.
```suggestion
	$path    = '/sites/' . rawurlencode( $blog_id ) . '/blogging-prompts?from=' . rawurlencode( $day_before ) . '&number=10&_locale=' . rawurlencode( $locale );
```

Reply:
🤦🏼 Working too quickly on that one--thanks for the correction.